src/redux/store.js

```diff
@@ -0,0 +1,13 @@
+import { configureStore } from "@reduxjs/toolkit";
+import userReducer from "./reducers/usersSlice";
+
+const reducer = {
+  users: userReducer,
+};
+
+const store = configureStore({
+  reducer: reducer,
+  devTools: true,
+});
+
+export default store;
```

Comment:
Went through all code, couldn't find any improvement suggestions. code is clean and modularized good to merge